app/src/main/java/com/example/kursaa/core/base/BaseViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.kursaa.core.base
+
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.kursaa.core.data.UiState
+import com.hadilq.liveevent.LiveEvent
+
+open class BaseViewModel : ViewModel(), DefaultLifecycleObserver {
+
+    private val _message by lazy { LiveEvent<String>() }
+    val message: LiveData<String> = _message
```

Comment:
Jeśli `_message` jest inicjowany `by lazy` to property `message` też powinno być inicjowane w ten sam sposób. To samo tyczy się `uiState`.